documentation/api/definitions.md

```diff
@@ -87,6 +87,49 @@ Enumeration that describes the current state of the collection rule.
 | `Throttled` | Indicates that the collection rule is temporarily throttled because the ActionCountLimit has been reached within the ActionCountSlidingWindowDuration. |
 | `Finished` | Indicates that the collection rule has completed and will no longer trigger. |
 
+## CapturedMethod
+
+First Available: 9.0 Preview 3
```

Comment:
reminder to update these `First Available: 9.0 Preview 3` with relevant version once the code has been approved